source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOBaseActionElement.h

```diff
@@ -17,7 +17,7 @@ typedef NS_ENUM(NSInteger, ACRActionType) {
     ACRSubmit,
     ACROpenUrl,
     ACRToggleVisibility,
-    ACRUnknownAction,
+    ACRUnknownAction = 6,
```

Comment:
>nknownAction = 6, [](start = 8, length = 17)

Can you explain this a bit?

Reply:
shared model enums and ios enums are matched in terms of integer value. shared model enum has an enum called CustomAction that's not supported in ios so it was not included in ios, and to able to match the integer value, 6 was assigned to ACRUnknownAction to ensure that the value matches with shared model enum. I added a unit test to make they works in the future.

---
In reply to: [336631355](https://github.com/microsoft/AdaptiveCards/pull/3526#discussion_r336631355) [](ancestors = 336631355)

Reply:
It would be good to have comments noting that this enum must be kept in sync with Shared::ActionType and explaining the missing enum value.

---
In reply to: [336674895](https://github.com/microsoft/AdaptiveCards/pull/3526#discussion_r336674895) [](ancestors = 336674895,336631355)

Reply:
thank you; will add it.

---
In reply to: [338165281](https://github.com/microsoft/AdaptiveCards/pull/3526#discussion_r338165281) [](ancestors = 338165281,336674895,336631355)